ampersand-view.js

```diff
@@ -137,13 +146,13 @@ _.extend(View.prototype, {
     remove: function () {
         var parsedBindings = this._parsedBindings;
         if (this.el && this.el.parentNode) this.el.parentNode.removeChild(this.el);
-        if (this._subviews) _.chain(this._subviews).flatten().invoke('remove');
```

Comment:
Changed from using `chain`, as lodash's `chain`/base wrapper functionality is not available as a module. I figured it wasn't worth importing the whole lodash library for the ability to chain for one line, and it wasn't worth using the flattened array's `forEach` method, and then calling `invoke` on each subview passed.